test/components/views/rooms/RoomTile-test.tsx

```diff
@@ -79,12 +87,44 @@ describe("RoomTile", () => {
     let room: Room;
     let renderResult: RenderResult;
     let sdkContext: TestSdkContext;
+    let showMessagePreview = false;
 
     filterConsole(
         // irrelevant for this test
         "Room !1:example.org does not have an m.room.create event",
     );
 
+    const addMessageToRoom = (ts: number) => {
+        const message = mkMessage({
+            event: true,
+            room: room.roomId,
+            msg: "test message",
+            user: client.getSafeUserId(),
+            ts,
+        });
+
+        room.timeline.push(message);
+    };
+
+    const addThreadMessageToRoom = (ts: number) => {
+        const message = mkMessage({
+            event: true,
+            room: room.roomId,
+            msg: "test thread reply",
+            user: client.getSafeUserId(),
+            ts,
+        });
+
+        // Mock thread reply for tests.
+        jest.spyOn(room, "getThreads").mockReturnValue([
+            // @ts-ignore
```

Comment:
If anyone knows how to properly add a thread in a more „integration test“ fashioned way, please leave a comment.